extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/devui/SchedulerJsonRPCService.java

```diff
@@ -26,6 +26,7 @@
 import io.quarkus.vertx.core.runtime.context.VertxContextSafetyToggle;
 import io.smallrye.common.vertx.VertxContext;
 import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
```

Comment:
I don't believe that ny of these method need to be Uni, they can just be @NonBlocking.